src/logging.h

```diff
@@ -94,6 +97,13 @@ namespace BCLog {
         /** Log categories bitfield. */
         std::atomic<uint32_t> m_categories{0};
 
+        //! Category-specific log level. Overrides the global `m_log_level`.
```

Comment:
nit: `m_log_level` is not global. `s/the global//`

Reply:
It's the global log level setting, so this seems fine. Can write  "global `m_log_level` setting" in one of the follow-ups to ensure no one thinks it means a global variable.

Reply:
Removed "the global"
```diff
-        //! Category-specific log level. Overrides the global `m_log_level`.
+        //! Category-specific log level. Overrides `m_log_level`.
```